include/jemalloc/internal/arena_inlines_b.h

```diff
@@ -65,6 +93,10 @@ arena_prof_info_get(tsd_t *tsd, const void *ptr, emap_alloc_ctx_t *alloc_ctx,
 	if (unlikely(!is_slab)) {
 		/* edata must have been initialized at this point. */
 		assert(edata != NULL);
+		if (reset_recent &&
+		    large_dalloc_safety_checks(edata, ptr, edata_szind_get(edata))) {
```

Comment:
nit: over 80-char max length. With the indent maybe:
```
		if (reset_recent && large_dalloc_safety_checks(edata, ptr,
		     edata_szind_get(edata))) {
```

Reply:
Ahh, sorry, had my editor set to 4-space tabs.